open-authoring/github-auth/validation.ts

```diff
@@ -0,0 +1,27 @@
+import { getForkName, getHeadBranch, setForkName } from "../open-authoring/repository"
+import { useCMS } from "tinacms"
+
+export const validate = async () => {
+    var validations = {
+        userIsAuthenticated: false,
+        forkIsValid: false
+    }
+
+    const cms = useCMS()
```

Comment:
hooks typically shouldn't be called in functions. Depending on how the function is called it could violate the rules of hooks and blow up: https://reactjs.org/docs/hooks-rules.html